lib/mayaHydra/hydraExtensions/sceneIndex/mayaHydraMayaUsdProxyShapeSceneIndex.cpp

```diff
@@ -0,0 +1,134 @@
+//
+// Copyright 2023 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "mayaHydraMayaUsdProxyShapeSceneIndex.h"
+
+#if defined(MAYAHYDRALIB_MAYAUSD_IS_USED_TO_BUILD)
```

Comment:
I would be more assertive and do:
```suggestion
#if not defined(MAYAHYDRALIB_MAYAUSD_IS_USED_TO_BUILD)
#error MAYAHYDRALIB_MAYAUSD_ENABLED must be defined!
#endif
```